tests/integration/wsapi/test_consumer.py

```diff
@@ -9,6 +9,7 @@
 from pydantic.error_wrappers import ValidationError
 
 from aap_eda.core import models
+from aap_eda.core.enums import CredentialType
```

Comment:
@hsong-rh can we add a test with system vault credential. I am hoping the label gets set when we add it in.

Reply:
Will add it later when db migrations are merged

Reply:
@mkanoor tests are added